### ISSUE TITLE
Sometimes macros return empty string

### ISSUE THREAD

**Issue by Ohrimenko1988** [eclipse-che/che#9321]
### Description
In rare cases, on the OCP platform, we have a situation when after command start,  output of the macros is not displayed.

This issue related to https://github.com/openshiftio/openshift.io/issues/2856

**Comment by Ohrimenko1988** [comment]
Can't reproduce on the 6.4.0 and 6.5.0 versions

**Comment by rhopp** [comment]
@Ohrimenko1988 Hi! This popped in my todo list, when somebody wanted to close the downstream issue (https://github.com/openshiftio/openshift.io/issues/2856), so I reserved half an hour to investigate this issue and I think I found a reliable way how to reproduce this issue:

1) Start workspace, switch to `Manage commands` view in the right panel
2) Make sure you have no files opened in editor.
3) Stop workspace & start it again - it should open in the same state, as it was stopped - the `Manage commands` panel should be visible (a.k.a. the `projects view` has not been "populated/shown")
4) Open any command that has `${current.project.path}` macro - it resolves as empty string.

I'll try to record screencast to show the steps on video.
